google/cloud/security/inventory/inventory_loader.py

```diff
@@ -198,13 +197,13 @@ def _send_email(cycle_timestamp, status, sendgrid_api_key,
     except EmailSendError:
         LOGGER.error('Unable to send email that inventory snapshot completed.')
 
-def main(unused_argv=None):
+def main():
```

Comment:
Better to accept argv and then delete it when not used, for compatibility with apputils and gflags.

def main(argv):
    """..."""
    del argv